x-pack/legacy/plugins/maps/server/maps_telemetry/maps_telemetry.js

```diff
@@ -88,24 +99,23 @@ export function buildMapsTelemetry(savedObjects, settings) {
   };
 }
 
-async function getSavedObjects(savedObjectsClient) {
-  const gisMapsSavedObject = await savedObjectsClient.find({
-    type: MAP_SAVED_OBJECT_TYPE
-  });
-  return _.get(gisMapsSavedObject, 'saved_objects');
+async function getMapSavedObjects(savedObjectsClient) {
+  const mapsSavedObjects = await savedObjectsClient.find({ type: MAP_SAVED_OBJECT_TYPE });
+  return _.get(mapsSavedObjects, 'saved_objects');
+}
+
+async function getIndexPatternSavedObjects(savedObjectsClient) {
+  const indexPatternSavedObjects = await savedObjectsClient.find({ type: 'index-pattern' });
+  return _.get(indexPatternSavedObjects, 'saved_objects');
 }
 
 export async function getMapsTelemetry(server, callCluster) {
   const savedObjectsClient = getSavedObjectsClient(server, callCluster);
-  const savedObjects = await getSavedObjects(savedObjectsClient);
+  const mapSavedObjects = await getMapSavedObjects(savedObjectsClient);
```

Comment:
I want to say there are occasions where we can't rely on there necessarily being a maps saved object (maybe during migration?). Should we short-circuit this here if `mapSavedObjects` comes back undefined?

Reply:
How could this come back `undefined`? Aren't we getting an empty array? I can always make that explicit in the getter.